lib/elix/brain/redis_store.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Elix.Brain.RedisStore do
+  @moduledoc """
+  A Redis back-end for the Brain which persists memory to disk.
+  """
+
+  @behaviour Elix.Brain.Store
+  @encoded_binary_prefix "binterm:"
+
+  def start_link do
+    redis_url = System.get_env("REDIS_URL") || "redis://"
```

Comment:
Pretty much. If part of your test setup clears the database, you don't want that to accidentally happen in production. It still can but you should roadblock as much as possible. And halt running when a step (configuration) has been skipped.